doc/port-selfupdate.1.txt

```diff
@@ -29,6 +29,11 @@ OPTIONS
     Only check for updates - and install if available - for MacPorts itself. Do
     not update the ports tree.
 
+*--migrate*::
```

Comment:
Does this go with selfupdate, like "port selfupdate --migrate"? Can you explain to me what's intended here?

Reply:
Okay, I got it from the description. This will be something limited to developers, right?

Reply:
Yes, this is only required to avoid trashing a developer's installation by overwriting it with an older version. I'd rather just error out and ask a developer to re-install base manually than breaking their installation.

Strictly speaking, it wouldn't be required to expose this flag to users; we could just have made it an internal flag. However, "reinstall but never downgrade" may be a valid use-case for users, too, and this makes documenting the flag easier, so I just went ahead and exposed it.